### PR TITLE
Create $uri

### PR DESCRIPTION
**Pull request by Rakejs**
## Type
- [ ] Bug Fix
- [X] Functions: \ $uri[decode/encode;text]
encodes or decodes a query for proper request
- [ ] Callbacks: \<Callback Name>
- [ ] Handlers: \<Handler Type>
- [ ] Others: \______

Dependencies (Third Party Modules) needed: <Name | Github Link> (Maximum: 20MB~ size)

Want a credit? Discord tag or other social media link: RAKE#6882

Referenced Issue: #NaN (Answer for an issue, if any)

## Description
Pull Request Description Here

### CODE REVIEW

**Comment by XzFirzal** [comment]
It encode/decode text into encoded/decoded uri component?

**Comment by Rakejs** [comment]
> It encode/decode text into encoded/decoded uri component?

yea correct
like "hello world" would be encoded "hello%20world"

File: package/functions/funcs/uri.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module.exports = async (d) => {
+  const code = d.command.code;
+  const inside = d.unpack();
+  const err = d.inside(inside);
+  if (err) return d.error(err);
+  let [type,text=""] = inside.splits; 
+  if(!type) d.error(':x: Pls provide a Type(encode/decode)')
```
Comment (XzFirzal) [inline package/functions/funcs/uri.js:7]: Provide 'return' keyword, so code won't be continously executed after error

Comment (Rakejs) [reply package/functions/funcs/uri.js]: okay done forgot it sry

**Comment by Kino7916** [comment]
You might want to change `Pls` to `Please` and minimize `return` usages

**Pull request merged**